sql/item_mods.sql

```diff
@@ -25531,7 +25563,15 @@ INSERT INTO `item_mods` VALUES (22076,25,20);     -- Accuracy+20
 INSERT INTO `item_mods` VALUES (22076,346,5);     -- Avatar Perpetuation-5
 INSERT INTO `item_mods` VALUES (22077,23,25);     -- was_+1 Attack+25
 INSERT INTO `item_mods` VALUES (22077,25,25);     -- Accuracy+25
-INSERT INTO `item_mods` VALUES (22077,346,7);     -- Avatar Perpetuation-7
+INSERT INTO `item_mods` VALUES (22077,346,-7);     -- Avatar Perpetuation-7
```

Comment:
https://github.com/DarkstarProject/darkstar/blob/11cf09da0b81f199ee9abf4fc82d9bb28d20cad9/src/map/status_effect_container.cpp#L1425
looks like that expects positive values?